src/routes/assistants/+page.svelte

```diff
@@ -13,6 +14,7 @@
 	import Pagination from "$lib/components/Pagination.svelte";
 
 	export let data: PageData;
+	export let user: LayoutData["user"];
```

Comment:
user is already in Pagedata (data.user)?

Reply:
handled in https://github.com/huggingface/chat-ui/pull/773/commits/f978c9da6322e6487088db9b851df16d3824820e